news/2024/2024-04-09-skinning-contest-unchained.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Skinning Contest: Unchained"
+date: 2024-04-09 15:00:00 +0000
+---
+
+Unshackle your creativity and showcase it to its fullest by participating in the newest skinning contest!
+
+![](/wiki/shared/news/2024-04-09-skinning-contest-unchained/banner.jpg)
+
+The skinship team have thrown all caution to the wind and unlocked the ultimate skinning contest theme.... which is that there isn't one. Yes, you heard that right!
+
+With a throwback to the [1st skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement) the skinning community will once again be able to choose from a huge variety of topics and themes, whether it'd be anime girls, video games that heavily involve anime girls, online personalities that are themselves anime girls, or perhaps a skin that doesn't include any anime at all (don't worry, we won't judge you).
```

Comment:
> we also should expand that its not just the theme that is relaxed, but also the permission rules

(^quote from skinship staff chat)
I currently see no mention of this in this section at all

Reply:
76dd24902877a115a3ca529ed69c46a0d4cc8f65 sorted here